lib/compat/wordpress-6.2/class-gutenberg-rest-pattern-directory-controller-6-2.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Pattern_Directory_Controller_6_2 class
+ *
+ * @package    Gutenberg
+ * @subpackage REST_API
+ */
+
+/**
+ * Controller which provides REST endpoint for block patterns from wordpress.org/patterns.
+ */
+class Gutenberg_REST_Pattern_Directory_Controller_6_2 extends Gutenberg_REST_Pattern_Directory_Controller_6_0 {
```

Comment:
Should  we also update `get_collection_params` that unsets [per_page and page](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-pattern-directory-controller.php#L313), no? We could probably use the `rest_pattern_directory_collection_params` filter.

Reply:
Is it preferred to use a filter or override the parent method? Given we'll probably need to add other collection params too - for the `allowedBlocks`/`blockTypes` issue, at least.

Reply:
If we can do something with a filter, I think it's better to do it that way - less code.

Reply:
Added in https://github.com/WordPress/gutenberg/pull/45293/commits/6bfc3724326b5fc3e1414e732dae5c7383f7b7da